server.go

```diff
@@ -64,6 +64,7 @@ type SSServer struct {
 }
 
 func (s *SSServer) startPort(portNum int) error {
+	timeout := 59 * time.Second
```

Comment:
nit: this could be a file constant, ie. `tcpReadTimeout`

Reply:
Is this what you mean:
`const tcpReadTimeout time.Duration = 59 * time.Second`
at top of server.go?

Reply:
Yeah, looks good!